docs/source/installation.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight:: shell
+
+Installation
+============
+
+Stable release
+--------------
+
+To install :code:`py-libp2p`:, run this command in your terminal:
```

Comment:
There's no right or wrong when it comes to the [*narrative perspective*](https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/documentation.md#narrative-perspective) but we chose to use the *first-person "we" form* which means there is rarely ever a place for "you" in our docs.

E.g. see [Trinity's quickstart](https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html) written in *first person "we" form*